activities.json

```diff
@@ -166,3 +166,31 @@
 {"activity":"Learn about a distributed version control system such as Git","accessibility":0.0,"type":"education","participants":1,"price":0.0,"link":"","key":"9303608","enabled":true}
 {"activity":"Write a note of appreciation to someone","accessibility":0.0,"type":"social","participants":1,"price":0.0,"link":"","key":"1770521","enabled":true}
 {"activity":"Draw something interesting","accessibility":0.0,"type":"recreational","participants":1,"price":0.0,"link":"","key":"8033599","enabled":true}
+{"activity":"Learn and play a new card game","accessibility":0.0,"type":"recreational","participants":1,"price":0.0,"link":"https://www.pagat.com","key":"9660022","enabled":true}
+{"activity":"Write a list of things you are grateful for","accessibility":0.0,"type":"relaxing","participants":1,"price":0.0,"link":"","key":"2062010","enabled":true}
+{"activity":"Organize one of your collections","accessibility":0.0,"type":"busywork","participants":1,"price":0.0,"link":"","key":"6098037","enabled":true}
```

Comment:
What did you mean by collections?

Reply:
I mean like a collection like books, games, stamps, etc. Are you OK with an "(e.g., ...)" clause?

Reply:
Ah I understand. Personally, I've avoided that for simplicity's sake and I'd prefer to keep it consistent. Maybe we can split this into "Organize the books on your bookshelf", "Organize a cluttered drawer", etc?

Reply:
Works for me! Will push new commit later.